installer/charts/tssc-integrations/templates/acs/pod.yaml

```diff
@@ -129,10 +129,6 @@ spec:
         - bash
         - -c
         - "echo 'No op: Success'"
-      requests:
```

Comment:
I think it should be moved to the right place instead of being deleted, otherwise I believe it makes Snyk or SonarQube unhappy.